openbas-api/src/test/java/io/openbas/scheduler/jobs/ScenarioExecutionJobTest.java

```diff
@@ -77,12 +77,11 @@ void given_cron_in_one_minute_should_create_simulation() throws JobExecutionExce
 
     Scenario scenario = ScenarioFixture.getScenario();
     int minuteToStart = (zonedDateTime.getMinute() + 1) % 60;
+    int hourToStart = zonedDateTime.getHour() + ((zonedDateTime.getMinute() + 1) / 60);
+    hourToStart = hourToStart % 24;
+
     scenario.setRecurrence(
-        "0 "
-            + minuteToStart
-            + " "
-            + zonedDateTime.getHour()
-            + " * * *"); // Every day now + 1 minute
+        "0 " + minuteToStart + " " + hourToStart + " * * *"); // Every day now + 1 minute
     Scenario scenarioSaved = this.scenarioService.createScenario(scenario);
```

Comment:
The message is correct if minuteTostart is 0 (cron : 0 0 * * *)?

Reply:
I am not sure to understand what you say